vault/testing.go

```diff
@@ -49,7 +49,7 @@ import (
 	"github.com/hashicorp/vault/vault/cluster"
 	"github.com/hashicorp/vault/vault/seal"
 	"github.com/mitchellh/copystructure"
-	"github.com/mitchellh/go-testing-interface"
+	testing "github.com/mitchellh/go-testing-interface"
```

Comment:
I think this is causing the circle-ci failures, and i seem to be able to replicate those locally.